README.md

```diff
@@ -120,6 +120,18 @@ letsencrypt::certonly { 'foo':
 }
 ```
 
+#### Nginx authenticator
+
+To request a certificate for `foo.example.com` and `bar.example.com` with the
+`certonly` installer and the `nginx` authenticator:
+
+```puppet
+letsencrypt::certonly { 'foo':
+  domains => ['foo.example.com', 'bar.example.com'],
+  plugin  => 'nginx',
+}
+```
```

Comment:
Should this be an `@example Nginx authenticator` tag in `certonly.pp`? That way it ends up in `REFERENCE.md`. Not a blocker for this, but perhaps this should be done for all examples above

Reply:
They are in the README.md right now, but I tried to add them in the `certonly.pp` to. Please have a look at https://github.com/voxpupuli/puppet-letsencrypt/pull/267/commits/576948a1ff73f70af8eb7e38bd06241fe985ffea

Reply:
Shall I remove some of them from the README.md or keep them in both places? It feels somewhat redundant to me, to have all examples in both files.

Reply:
In most repos we've removed them. You can keep a general section that says where they can be found in the reference for better visibility.

Reply:
In the README.md they have some comments (explanations). How do you add comments to examples in Puppet Strings?
In the past I think I just did:
```
# @example asdf
#   # explanation
#   code
#   ...
```

Reply:
I also used Puppet code comments in the same way. I'm not aware of any other method.

Reply:
How do you like it now?
For some reason, I don't understand for what reason, the pipeline fails... Do you have an idea?